### ISSUE TITLE
Crazy ideas

### ISSUE THREAD

**Issue by Joncy** [netbeast-co/xy-deamon#1]
Could we integrate WIPS (+enhancements) onto it?? Might be able to do some cool stuff with that.

**Comment by jesusdario** [comment]
I feel it is better as a separated piece of software running, since this code most be running 100% of the time and must be secure. I leave the issue open to discuss it further and reading what enhancements are proposed.

(Plus WIPS is yet young since it needs to connect to a portable emitting device, like an Android Hotspot)

**Comment by Joncy** [comment]
Yes, I actually said the contrary to what I mean. I mean, building a location module that contains this daemon as well as WIPS to detect the position of other xways (since they can operate generating a subnet and connecting to one and since network SSID naming conflict was resolved by Luis). Just crazy ideas for future, I'll put 50c into the jar :)
